libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -14,12 +27,37 @@
 </template>
 
 <script lang="ts" setup>
-defineProps<{
-  src?: string
-  alt?: string
-  original: boolean
-  isDetail?: boolean
-}>()
+import { useImage } from '@vueuse/core'
+
+const props = withDefaults(
+  defineProps<{
+    src?: string
+    alt?: string
+    original: boolean
+    isDetail?: boolean
+    placeholder?: string
+  }>(),
+  {
+    src: '',
+    alt: '',
+    placeholder: '/placeholder.webp',
+  }
+)
+
+let cfImage = computed(() => {
+  if (props.src) {
+    return (
+      props.src.replace(
+        /https:\/\/image(-beta)?.w.kodadot.xyz\/ipfs\//,
+        'https://imagedelivery.net/jk5b6spi_m_-9qC4VTnjpg/'
+      ) + '/public'
+    )
+  }
+
+  return ''
```

Comment:
What's this? can we do without empty return?
is this a good idea to add vue-use in `libs/ui` for me it's supposed to be a bare component